README.md

```diff
@@ -74,6 +74,41 @@ webpi_product 'WindowsAzurePowerShellGet' do
 end
 ```
 
+
+### webpi_application
+
+#### Actions
+
+- :install: install an application using WebpiCmdLine
+
+#### Attribute Parameters
+
+- app_id: name attribute. Specifies the ID of an application to install.
```

Comment:
name attribute should be name property

Reply:
I have done it in a way similar to webpi_product.

Reply:
How exactly do you think should I change it?

Reply:
The naming convention changed in Chef 12.5 so you don't need to update the code. It's just a documentation thing. I can change it everywhere.